### DISCUSSION TITLE
Is this integration/maintainer still active

### DISCUSSION THREAD

**Discussion by tuxianerDE** [B5r1oJ0A9G/teufel_raumfeld#48] [General]
Hey just an open question (and ask for an honest answers). Is this package/integration for raumfeld under further development? I do know and am fully aware of that this is all just a hobby for most of us, however there people using this plugin and just wanted to check what the current state of affairs is?

I see that almost like a year no real development or issue topic has been adressed anymore.

Cheers

**Comment by B5r1oJ0A9G** [comment]
Hi there!

I've been too busy with other things lately. I don't think that will change soon either. I wouldn't say that the project has stopped, but the time frame is very limited.

Community support for further development is of course very welcome!

**Reply by tuxianerDE** [reply]
All good just seing all the updates that get pushed via HACS really appreciate it and to be honest I did not intent to blame/push you here rather a question coz pitty to llose this nice integration

**Comment by mathebox** [comment]
Hi!
I also noticed that the development of the integration is a bit stale. As mentioned above, I also know the challenge of maintaining a hobby project in your spare time. And it would be a pity if we would lose this nice integration at some point.
Therefore, I would like to offer my support to the project. I would be happy to take over maintenance tasks and investigate bugs (if possible). However, just creating PRs and waiting for a response isn't rewarding in itself.  Would it be possible for me to get additional permissions for the project to speed up development?

**Reply by B5r1oJ0A9G** [reply]
Hey @mathebox!

I really appreciate your offer to help with this project! I'm also very keen to keep it alive, so it's great to have your support.

I have now added you as a collaborator. This will allow you to get involved, tackle issues and help move things forward. I look forward to working with you and achieving more!

Thanks again for your willingness to support!
